src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -156,6 +156,7 @@
 			<Choose>
 				<When Condition="'$(Configuration)'=='Debug'">
 					<PropertyGroup>
+						<ProvisioningType>manual</ProvisioningType>
```

Comment:
Are the tabs good here on github it shows all messed up

Reply:
Seems good to me 
![image](https://github.com/nventive/UnoApplicationTemplate/assets/17461593/535ac421-fae5-43a7-a043-11e56882fb8e)